docs/pii-config/selectors.md

```diff
@@ -1,3 +1,82 @@
 # PII Selectors
 
-This section is documented as part of Sentry's docs: [Selectors for Advanced Data Scrubbing](https://docs.sentry.io/data-management/advanced-datascrubbing/#selectors).
+You have the possibility to select a region of the event using JSON-path-like syntax. To delete a specific key in "Additional Data", you could for example use:
+
+```json
+{
+  "applications": {
+    "extra.foo": ["@anything:remove"]
+  }
+}
+```
+
+## Boolean logic
+
+You can combine selectors using boolean logic.
+
+* Prefix with `!` to invert the selector. `foo` matches the JSON key `foo`, while `(~foo)` matches everything but `foo`.
+* Build the conjunction (AND) using `&&`, such as: `foo && !extra.foo` to match the key `foo` except when inside of `extra`.
+* Build the disjunction (OR) using `||`, such as: `foo || bar` to match `foo` or `bar`.
+
+## Wildcards
+
+* `**` matches all subpaths, so that `foo.**` matches all JSON keys within `foo`.
+* `*` matches a single path item, so that `foo.*` matches all JSON keys one level below `foo`.
+
+## Value types
+
+The following can be used to select subsections by JSON-type or semantic meaning.
+
+* `$string`
+* `$number`
+* `$boolean`
+* `$datetime`
+* `$array`
+* `$object`
+* `$event`
```

Comment:
Can you group those into the generic ones and the schema-specific ones?

Reply:
Done and also synced with https://github.com/getsentry/sentry-docs/pull/1562